src/Encryption/Codec/EncryptionCodec.cs

```diff
@@ -8,20 +8,20 @@ namespace TemporalioSamples.Encryption.Codec;
 public sealed class EncryptionCodec : IPayloadCodec, IDisposable
 {
     public const string DefaultKeyID = "test-key-id";
-    public static readonly byte[] DefaultKey = System.Text.Encoding.ASCII.GetBytes("test-key-test-key-test-key-test!");
+    private static string DefaultKey = "test-key-test-key-test-key-test!";
```

Comment:
Should be no need to change this I figure, let's keep keys as byte arrays from a user POV (they don't have to have string representations)